assets/product-info.js

```diff
@@ -135,18 +131,18 @@ if (!customElements.get('product-info')) {
           });
       }
 
-      getProductInfoUrl(url, variantId, shouldFetchFullPage = false) {
+      getSelectedVariant(productInfoNode) {
+        const selectedVariant = productInfoNode.querySelector('variant-selects [data-selected-variant]')?.innerHTML;
+        return !!selectedVariant ? JSON.parse(selectedVariant) : null;
+      }
+
+      getProductInfoUrl(url, optionValues, shouldFetchFullPage = false) {
         const params = [];
 
         !shouldFetchFullPage && params.push(`section_id=${this.sectionId}`);
 
-        if (variantId) {
-          params.push(`variant=${variantId}`);
-        } else {
-          const optionValues = this.variantSelectors.selectedOptionValues;
-          if (optionValues.length) {
-            params.push(`option_values=${optionValues.join(',')}`);
-          }
+        if (optionValues.length) {
+          params.push(`option_values=${optionValues.join(',')}`);
```

Comment:
This change isn't really meaningful. You can request a product and specify the selection either with a variant or with option value IDs, you get the same response either way. The reason for this change is because we don't want to rely on a variant here anymore.